google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionImpl.java

```diff
@@ -340,10 +343,11 @@ public void run() {
     return res;
   }
 
-  TransactionContextImpl newTransaction() {
+  TransactionContextImpl newTransaction(Options options) {
```

Comment:
Can this be just `TransactionOption` instead of `Options`?

Reply:
That would be possible (assuming you mean `TransactionOption...`), but I would prefer to keep this as the `Options` object. This makes it a required argument, making it less likely to be missed by accident as it will cause a compile error if you don't supply an `Options` object. If we change it to `TransactionOption...` it would not cause a compile time error if we forget to pass in the options somewhere.

Reply:
Hmm, makes sense.